pandas/tests/scalar/test_nat.py

```diff
@@ -123,6 +123,11 @@ def test_round_nat(klass):
             assert round_method(freq) is ts
 
 
+def test_timestamp():
```

Comment:
pls just add this where the others are (IOW in the list), don't special case this.